package.json

```diff
@@ -40,6 +40,7 @@
 		"@bitrise/steplib-search": "^2.3.0",
 		"@datadog/browser-logs": "^1.7.0",
 		"@datadog/browser-rum": "^2.6.1",
+		"@types/segment-analytics": "0.0.34",
```

Comment:
Do we need the type definition file? Only js files were modified so we are not using any types.
(But for me it can stay here, just asking)

Reply:
I went with the TypeScript version based on the docs I saw on WFE gradually moving to React and TypeScript. In the Monolith, we use Segment in both JS and TS files, so I assumed that may happen here as well.